manager.go

```diff
@@ -422,6 +423,24 @@ func (m *Manager) dial(ctx context.Context) error {
 	return nil
 }
 
+// process non-subsystem effects of a config (i.e. agent-specific stuff that needs to happen when loading cache and when updating).
+func (m *Manager) processConfig(cfg map[string]*pb.DeviceSubsystemConfig) {
+	if agent, ok := cfg["viam-agent"]; ok {
+		if debug, ok := agent.GetAttributes().AsMap()["debug"].(bool); !ok {
+			m.logger.Error("viam-agent debug attribute is present but is not a bool")
```

Comment:
nit: Not a correct error message. This returns false (!ok) if it's absent OR not a bool. You can cast an empty interface to bool essentially.

Reply:
oops, good catch, fixed in 8ba8315